go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	//
 	// when updating k8s.io/kubernetes, make sure to update the replace section too
 	//
-	k8s.io/kubernetes v1.25.3
+	k8s.io/kubernetes v1.25.4
```

Comment:
as mentioned in the comment above, the `replace` section needs updating too

Reply:
@yati1998 @nixpanic this should address the issue https://github.com/ceph/ceph-csi/pull/3551